target/linux/ipq806x/image/Makefile

```diff
@@ -316,4 +316,21 @@ define Device/zyxel_nbg6817
 endef
 TARGET_DEVICES += zyxel_nbg6817
 
+define Device/askey_rt4230w
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+        DEVICE_VENDOR := Askey
+        DEVICE_MODEL := RAC2V1K/RT4230W
+        DEVICE_VARIANT := rev6
+        SOC := qcom-ipq8065
+        BLOCKSIZE := 128k
+        PAGESIZE := 2048
+        BOARD_NAME := rt4230w
```

Comment:
That's probably not required.

Reply:
I left it because deleting it broke installation